pkg/deploy/deploy.go

```diff
@@ -291,10 +289,11 @@ func (c *componentDeployer) removeChildren(ctx context.Context, parent, root gra
 
 		// after the first iteration
 		if parent != root {
-			l.WithFields(field.F("root", root.Config.Coordinate)).
+			l.WithFields(field.F("root", root.Config.Coordinate), field.StatusDeploymentSkipped()).
```

Comment:
Pls add the field to L288 :) 
The custom logger there already has a lot of common attributes between the two log lines :)